pkg/nodeconfig/nodeconfig.go

```diff
@@ -111,6 +113,21 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// GetClusterDNS returns the IP address of the DNS server, given the service CIDR of the cluster
+func GetClusterDNS(iprange string) (string, error) {
```

Comment:
I suggest moving this into the `cluster` package

Reply:
fixed in https://github.com/openshift/windows-machine-config-operator/pull/777/commits/6daac5ef10b9c23a127ec7757b69966d21507170